src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTapeCorner.java

```diff
@@ -284,56 +229,110 @@ public AxisAlignedBB getBoundingBox(final IBlockState state, final IBlockAccess
-                                      (float) W_START_COLLISION_Z,
-                                      (float) W_END_COLLISION_X,
-                                      (float) HEIGHT_COLLISION,
-                                      (float) W_END_COLLISION_Z);
+                    (float) BOTTOM_COLLISION,
+                    (float) W_START_COLLISION_Z,
+                    (float) W_END_COLLISION_X,
+                    (float) HEIGHT_COLLISION,
+                    (float) W_END_COLLISION_Z);
         }
         if (state.getValue(FACING).equals(SOUTH))
         {
             return new AxisAlignedBB((float) S_START_COLLISION_X,
-                                      (float) BOTTOM_COLLISION,
-                                      (float) S_START_COLLISION_Z,
-                                      (float) S_END_COLLISION_X,
-                                      (float) HEIGHT_COLLISION,
-                                      (float) S_END_COLLISION_Z);
+                    (float) BOTTOM_COLLISION,
+                    (float) S_START_COLLISION_Z,
+                    (float) S_END_COLLISION_X,
+                    (float) HEIGHT_COLLISION,
+                    (float) S_END_COLLISION_Z);
         }
         else
         {
             return new AxisAlignedBB((float) E_START_COLLISION_X,
-                                      (float) BOTTOM_COLLISION,
-                                      (float) E_START_COLLISION_Z,
-                                      (float) E_END_COLLISION_X,
-                                      (float) HEIGHT_COLLISION,
-                                      (float) E_END_COLLISION_Z);
+                    (float) BOTTOM_COLLISION,
+                    (float) E_START_COLLISION_Z,
+                    (float) E_END_COLLISION_X,
+                    (float) HEIGHT_COLLISION,
+                    (float) E_END_COLLISION_Z);
         }
     }
 
-    // =======================================================================
-    // ======================= Rendering & IBlockState =======================
-    // =======================================================================
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @SideOnly(Side.CLIENT)
+    @Override
+    @Deprecated
+    public boolean shouldSideBeRendered(final IBlockState blockState, final IBlockAccess blockAccess, final BlockPos pos, final EnumFacing side)
+    {
+        return true;
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @Deprecated
+    @Nullable
+    @Override
+    public AxisAlignedBB getCollisionBoundingBox(
+            final IBlockState blockState, final World worldIn, final BlockPos pos)
+    {
+        return null;
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @Override
+    @Deprecated
+    public boolean isOpaqueCube(final IBlockState state)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)